lib/screens/pairing/pair_screen.dart

```diff
@@ -107,6 +114,9 @@ class _PairState extends State<PairScreen> {
   }
 
   void _showError(Object error, BackupSource source) {
+    FirebaseCrashlytics.instance.log(
+        'Error recovering from credentionals: $error ${error is CredentialException ? 'CredentialException Code : ${error.code}, CredentialException Messsage: ${error.message}, ${error.details}' : ''}');
```

Comment:
This condition check repeats many times, we should create a utility function to make code less verbose